SensusUI/Inputs/NumberSliderInput.cs

```diff
@@ -172,48 +240,69 @@ public override View GetView(int index)
                     {
                         _incrementalValue = newIncrementalValue;
                         _incrementalValueHasChanged = true;
-                        _sliderLabel.Text = GetLabelText(index) + "  " + _incrementalValue;
+                        _sliderLabel.Text = DisplaySliderValue ? GetLabelText(index) + "  " + _incrementalValue : GetLabelText(index) + "  " + _tipText;
```

Comment:
Use private variables (`_displaySliderValue`) within the class rather than properties.